### PR TITLE
Minor tweak to cover alt text

### LINKED ISSUE #837: Minor tweak to cover alt text
At:
https://github.com/internetarchive/openlibrary/blob/78219a4e2d48584246617fd682088b58be06841a/openlibrary/templates/covers/book_cover_work.html#L13

it would help to insert a comma into the $title $_('by')  for better legibility, or to ease into i18n, lose the 'by' and just use a space wrapped foreslash, so $title $_(' / ')

### PR DESCRIPTION
**Pull request by salman-bhai**
Closes #837

### CODE REVIEW

**Comment by salman-bhai** [comment]
@LeadSongDog looks good?

**Comment by LeadSongDog** [comment]
LGTM

**Comment by salman-bhai** [comment]
@mekarpeles

File: openlibrary/templates/covers/book_cover.html [hunk 1]
```diff
@@ -14,7 +14,7 @@
 $if size == "M":
     <div class="coverMagic">
             <div class="SRPCover bookCover" style="display: $cond(cover_url, 'block', 'none');">
-                <a href="$cover_lg" class="coverLook" title="Pull up a bigger book cover"><img itemprop="image" src="$cover_url" class="cover" alt="Cover of: $title $_('by') $author_names"/></a>
+                <a href="$cover_lg" class="coverLook" title="Pull up a bigger book cover"><img itemprop="image" src="$cover_url" class="cover" alt="Cover of: $title $_(' | ') $author_names"/></a>
```
Comment (hornc) [inline openlibrary/templates/covers/book_cover.html]: this doesn't need the underscore now, there is no reason to pass the character to python if you just want the character.

`alt="Cover of: $title | $author_names"`
will do the same thing

**Comment by salman-bhai** [comment]
Anything else @hornc ?

**Comment by mekarpeles** [comment]
lgtm! merged

**Comment by LeadSongDog** [comment]
Woohoo!!! Bonus: as of this morning, covers are again being served from OL vice IA, e.g. https://covers.openlibrary.org/b/id/4175446-M.jpg

**Pull request merged**